### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by jenny-wild**
test  test

### CODE REVIEW

**Comment by jenny-wild** [comment]
nice

**Comment by jenny-wild** [comment]
very nice

File: index.js [hunk 1]
```diff
@@ -2,4 +2,3 @@
 // how does this work again?
 
 // whatever you want
-// comment again
```
Comment (ellenmarek) [inline index.js:5]: I see what you did there

**Review by ellenmarek — COMMENTED** [review]
LGTM

**Pull request merged**